tests/integration/integration_test.py

```diff
@@ -939,6 +746,157 @@ def my_on_revoke(consumer, partitions):
     c.close()
 
 
+def verify_avro():
+    base_conf = {'bootstrap.servers': bootstrap_servers,
+                 'error_cb': error_cb,
+                 'api.version.request': api_version_request,
+                 'api.version.fallback.ms': 0,
+                 'broker.version.fallback': '0.11.0.0',
+                 'schema.registry.url': schema_registry_url}
+
+    consumer_conf = dict(base_conf, **{
+        'group.id': 'test.py',
+        'session.timeout.ms': 6000,
+        'enable.auto.commit': False,
+        'on_commit': print_commit_result,
+        'default.topic.config': {
```

Comment:
we should move away from default.topic.config, it will be deprecated in the release this is in (v1.0.0)